pkg/cli/config/config_test.go

```diff
@@ -0,0 +1,61 @@
+package config
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestGetBackplaneConfig(t *testing.T) {
+	t.Run("it returns the user defined proxy instead of the configuration variable", func(t *testing.T) {
+
+		svr := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			_, _ = w.Write([]byte("dummy data"))
+		}))
+
+		userDefinedProxy := "example-proxy"
+		t.Setenv("BACKPLANE_URL", svr.URL)
+		t.Setenv("HTTPS_PROXY", userDefinedProxy)
+		config, err := GetBackplaneConfiguration()
+		if err != nil {
+			t.Error(err)
+		}
+
+		if config.ProxyURL != nil && *config.ProxyURL != userDefinedProxy {
+			t.Errorf("expected to return the explicitly defined proxy %v instead of the default one %v", userDefinedProxy, config.ProxyURL)
+		}
+	})
+
+}
+
+func TestGetBackplaneConnection(t *testing.T) {
+	t.Run("should fail if backplane API return connection errors", func(t *testing.T) {
+
+		svr := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			_, _ = w.Write([]byte("dummy data"))
+		}))
+
+		proxyURL := "http://squid.myproxy.com"
+		t.Setenv("BACKPLANE_URL", svr.URL)
+		t.Setenv("HTTPS_PROXY", proxyURL)
+		config, err := GetBackplaneConfiguration()
+		if err != nil {
+			t.Error(err)
+		}
+
+		err = config.CheckAPIConnection()
+		if err != nil {
+			t.Failed()
+		}
+
+	})
+
+	t.Run("should fail for empty proxy url", func(t *testing.T) {
+		config := BackplaneConfiguration{URL: "https://api-backplane.apps.openshiftapps.com", ProxyURL: nil}
```

Comment:
Must've gotten in there by mistake, but as you already know we shouldn't be even hinting to the API URL which remotely resembles the actual API.

```suggestion
		config := BackplaneConfiguration{URL: "https://dummy-url", ProxyURL: nil}
```